sycl/include/CL/sycl/handler.hpp

```diff
@@ -286,6 +286,22 @@ class handler {
       case access::target::local: {
         detail::LocalAccessorBaseHost *LAcc =
             static_cast<detail::LocalAccessorBaseHost *>(Ptr);
+        // Stream implementation creates local accessor with size per work item
```

Comment:
Any reason to use local memory that is "partitioned" per WIs instead of just using private memory?

Reply:
We cannot allocate private memory of size which is known only in runtime, and private memory is usually in registers and probably it will be not good to store data there. We would need to make the flush buffer size a template argument.

Reply:
You are right, thx!